### ISSUE TITLE
Enable monitoring of multiple REDIS_CONNECTION_STRING

### ISSUE THREAD

**Issue by rohit-co3046** [chrnola/redis-streams-exporter#9]
Unable to add multiple REDIS_CONNECTION_STRING

**Comment by chrnola** [comment]
Yes, that is a current limitation. If you need multiple connection strings,
you can try running multiple instances of the app.

**Comment by rohit-co3046** [comment]
Ok. Thanks for the updates

**Comment by sksaranraj** [comment]
Hi Any plans to add this?

**Comment by chrnola** [comment]
@sksaranraj No. I suggest trying https://github.com/oliver006/redis_exporter instead. It is likely a better fit for your needs.
